src/client/Microsoft.Identity.Client/Http/LinearRetryPolicy.cs

```diff
@@ -21,8 +23,11 @@ public LinearRetryPolicy(int delayMilliseconds, int maxRetries, Func<HttpRespons
             _retryCondition = retryCondition;
         }
 
-        public bool pauseForRetry(HttpResponse response, Exception exception, int retryCount)
+        public bool PauseForRetry(HttpResponse response, Exception exception, int retryCount)
         {
+            // referenced in the unit tests
+            numRetries = retryCount + 1;
```

Comment:
where is the pause happening? or where is `DelayInMilliseconds` used during retries

Reply:
@gladjohn I don't believe I touch any of that functionality in this PR? Where was it being used and where did the pause happen previously?

Reply:
@Robbie-Microsoft I do not see any pauses anywhere in product code, searched both `Thread.Sleep` and `Task.Delay` and nothing in managed identity flows. maybe we never implemented it. I added a separate work item to track this - https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/5260

Reply:
you can resolve this conversation